caas/ecs/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package ecs_test
```

Comment:
Is there an import loop that prevents us putting this in the _ecs_ package and using exported suite methods instead of _export_test.go_

Reply:
We will moving stuff around in the following PRs so we can try changing this in one of those.